lib/charms/tls_certificates_interface/v1/tls_certificates.py

```diff
@@ -1,6 +1,7 @@
 # Copyright 2021 Canonical Ltd.
```

Comment:
Need to move this file into `/v2` and either keep the previous `v1` intact or delete it.

Reply:
I moved the new version to `/v2` and I've kept the previous one under `/v1`. Sorry about the Github diff 😅

Reply:
Humm I don't aggree that we need to use a new `v` for this change. For now v0/v1 were used for changes in the interface itself. The lib already supports major breaking changes with `LIBAPI` and I think we should leverage this.

Reply:
"in the interface itself" we mean the jsonschema? Is that right?

Reply:
@gruyaume not sure what you mean: this pr bumped LIBAPI to 2, so I think it's supposed to go into a `/v2` folder.
I think `charmcraft publish-lib` would error out if you attempt to publish `LIBAPI=2` from `/v1`.

Reply:
@sed-i yes, the LIBAPI value and the version in the path should match.

Reply:
My bad!